rockcraft/commands/init.py

```diff
@@ -52,6 +55,72 @@ def init(rockcraft_yaml_content: str) -> None:
 class InitCommand(AppCommand):
     """Initialize a rockcraft project."""
 
+    _INIT_TEMPLATES = {
```

Comment:
It's not needed here yet, but when this starts getting more complex maybe we can port Charmcraft's jinja-based implementation to a `TemplateService` in craft-application.